stack.yml

```diff
@@ -206,10 +206,10 @@ functions:
       write_debug: true
       read_debug: true
 
-  system-metrics:
+  metrics:
     lang: go
-    handler: ./system-metrics
-    image: functions/system-metrics:0.1.1
+    handler: ./metrics
+    image: functions/metrics:0.1.1
```

Comment:
Can we retain the image name since the new name doesn't exist yet, so it may break when merged?

Reply:
ah yes - excellent point.